src/util/ConfigManager/ConfigManager.cpp

```diff
@@ -321,7 +308,7 @@ ConfigManager::getListOfNotChangedConfigOptionsWithDefaultValuesAsString()
     const {
   // For only looking at the configuration options in our map.
   auto onlyConfigurationOptionsView = std::views::transform(
-      configurationOptions_, [](const auto& pair) { return pair.second; });
+      configurationOptions_, [](const auto& pair) { return *pair.second; });
```

Comment:
I think there is `std::views::values` which does exactly that (minus the pointer dereferencing, but I have a suggestion for this above).

Reply:
Done, see [b2e58d4](https://github.com/ad-freiburg/qlever/pull/1040/commits/b2e58d4ca902419ba4b6213529bc4d5f720319f0).